ej2-angular/datetimepicker/getting-started.md

```diff
@@ -1,14 +1,14 @@
 ---
 layout: post
-title: Getting started with Angular Datetimepicker component | Syncfusion
-description:  Checkout and learn about Getting started with Angular Datetimepicker component of Syncfusion Essential JS 2 and more details.
+title: Getting Started with Angular DateTime Picker Component | Syncfusion
+description: Learn here all about getting started with Syncfusion Essential Angular DateTime Picker component, it's elements and more.
```

Comment:
Revert this and remove unwanted space only.

Reply:
reverted